NEWS.md

```diff
@@ -4,6 +4,14 @@ Trixi.jl follows the interpretation of [semantic versioning (semver)](https://ju
 used in the Julia ecosystem. Notable changes will be documented in this file
 for human readability.
 
+## Changes in the v0.8 lifecycle
+
+#### Changed
+
+- The AMR routing for `P4estMesh` and `T8codeMesh` was changed to work on the product
```

Comment:
```suggestion
- The AMR routines for `P4estMesh` and `T8codeMesh` was changed to work on the product
```
Just a small change, but otherwise looks good.